city_scrapers/spiders/chi_ssa_69.py

```diff
@@ -0,0 +1,130 @@
+import json
+import re
+from datetime import datetime, timedelta
+
+from city_scrapers_core.constants import COMMISSION, COMMITTEE, FORUM, NOT_CLASSIFIED
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+
+
+class ChiSsa69Spider(CityScrapersSpider):
+    name = "chi_ssa_69"
+    agency = "Chicago Special Service Area #69 95th & Ashland Avenue"
+    timezone = "America/Chicago"
+
+    @property
+    def start_urls(self):
+        today = datetime.now()
+        last_week = today - timedelta(days=7)
+        in_two_months = today + timedelta(days=60)
+
+        # I think there is a problem where the timezone in Chicago will be
+        # changing during tis two month period
+        # This results in meeting times that look wrong (off by an hour)
+
+        from dateutil import tz
+
+        # determine the current GMT offset for Chicago time (varies throughout the year)
+        current_chicago_time = datetime.now(tz.gettz('America/Chicago'))
+        current_chicago_time_gmt_offset = str(current_chicago_time)[-6:]
+
+        return [(
+            "https://www.googleapis.com/calendar/v3/calendars/gagdcchicago%40gmail.com/events"
+            "?calendarId=gagdcchicago@gmail.com&singleEvents=true&timeZone={}&"
```

Comment:
If you substitute the string `America/Chicago` for the `timeZone` instead of inputting the offset it will always return the correct offset even if it shifts. Thanks for catching this though! I'll update the other scrapers using Google Calendar

Reply:
Cool. I made that change.